src/Microsoft.ML.TimeSeries/SequentialTransformerBase.cs

```diff
@@ -288,25 +357,41 @@ public IRowToRowMapper GetRowToRowMapper(Schema inputSchema)
             throw new InvalidOperationException("Not a RowToRowMapper.");
         }
 
-        public sealed class SequentialDataTransform : TransformBase, ITransformTemplate
+        public IRowToRowMapper GetStatefulRowToRowMapper(Schema inputSchema)
```

Comment:
>GetStatefulRowToRowMapper [](start = 31, length = 25)

You made the interfaces internal, but then forgot I think to make the interface implementations explicit.